mudlet-mapper.xml

```diff
@@ -7077,29 +7136,99 @@ function mmp.addWingsLusternia()
     end
   end
   --all the bubblixes!
-  if mmp.settings.screwdriver then
-    linkBubblix("the Facility", 10185, "touch screwdriver")
-  end
-  if mmp.settings.wheel then
-    linkBubblix("the Dramube Triangle", 10509, "touch wheel")
-  end
-  if mmp.settings.mud then
-    linkBubblix("Mucklemarsh", 9985, "touch mud")
-  end
-  if mmp.settings.snowglobe then
-    linkBubblix("the Great Spirit Tree", 10992, "shake snowglobe")
-  end
-  if mmp.settings.cookie then
-    linkBubblix("Crumkindivia", 9888, "touch cookie")
-  end
-  if mmp.settings.head then
-    linkBubblix("the Bubble of Bottledowns", 11811, "touch doll")
+  local bubblixes =
+    {
+      {
+        setting = "figurine",
+        item = "icebix",
+        room = 23187,
+        areas = {"Northern Icewynd Mountains", "Southern Icewynd Mountains"},
+      },
+      {setting = "twist", item = "diobix", room = 18203, areas = {"Mount Dio"}},
+      {setting = "snowglobe", item = "treebix", room = 10992, areas = {"The Great Spirit Tree"}},
+      {setting = "periscope", item = "aquabix", room = 36073, areas = {"Aquagoria"}},
+      {setting = "pebble", item = "lyraabix", room = 33825, areas = {"Lyraa Ey Rielys"}},
+      {setting = "shard", item = "xionbix", room = 12652, areas = {"the Workshop of Xion"}},
+      {setting = "cookie", item = "crumbix", room = 9888, areas = {"Crumkindivia"}},
+      {setting = "wheel", item = "drambix", room = 10509, areas = {"the Dramube Triangle"}},
+      {setting = "head", item = "bottlebix", room = 11811, areas = {"the Bubble of Bottledowns"}},
+      {setting = "icicle", item = "frostbix", room = 10457, areas = {"Frosticia"}},
+      {setting = "screwdriver", item = "facbix", room = 10185, areas = {"the Facility"}},
+      {setting = "mud", item = "mucklebix", room = 9985, areas = {"Mucklemarsh"}},
+      {
+        setting = "tibia",
+        item = "cankerbix",
+        room = 11600,
+        areas = {"the Cankermore Battlegrounds"},
+      },
+    }
+  local shortcutCreated = false
+  for _, bubblix in pairs(bubblixes) do
+    if mmp.settings[bubblix.setting] then
+      local command = "touch " .. bubblix.item
```

Comment:
This will create `touch treeblix` while before it was `shake snowglobe`. Is that correct?

Reply:
Yes! Relatively recently, they added in new aliases for all the different bubble travel items in the style of treebix, aquabix, etc -  While there are two variations of the tree of trees bubblix (an emerald leaf, available for purchase, and a snowglobe which was available to some older players for free), they both respond to "touch treebix". Shake snowglobe would only have worked with the snowglobe style one, and could get confused if you had other snowglobe items in your inventory.